src/test/java/uk/gov/justice/hmpps/prison/aop/connectionproxy/OracleConnectionAspectTest.kt

```diff
@@ -218,15 +196,15 @@ class OracleConnectionAspectTest {
       fun `should set Nomis context and schema`() {
         connectionAspect.openProxySessionIfIdentifiedAuthentication(pooledConnection)
 
-        verify(pooledConnection).prepareStatement(
-          check {
-            assertThat(it).contains("nomis_context.set_context('AUDIT_MODULE_NAME', 'MERGE')")
-            assertThat(it).doesNotContain("nomis_context.set_context('AUDIT_USER_ID'")
-            assertThat(it).doesNotContain("nomis_context.set_client_nomis_context(")
-          }
-        )
-        verify(pooledConnection).prepareStatement("ALTER SESSION SET CURRENT_SCHEMA=some default schema")
-        verify(pooledPreparedStatement, times(2)).execute()
+        verify(nomisConfigurer).setSuppressXtagEvents(any<ResettableContextConnection>())
+        verify(nomisConfigurer).setDefaultSchema(any<ResettableContextConnection>())
+      }
+
+      @Test
+      fun `should return resettable context connection`() {
+        val conn = connectionAspect.openProxySessionIfIdentifiedAuthentication(pooledConnection)
+
+        assertThat(conn).isExactlyInstanceOf(ResettableContextConnection::class.java)
```

Comment:
We are now returning a resettable context connection for XTag event suppressoing connections.